### PR TITLE
chore: scaletest: collect database metrics using prometheus-postgres-exporter 

### PR DESCRIPTION
**Pull request by johnstcn**
Also fixes an inadvertent breakage to the script from formatting.

### CODE REVIEW

File: scaletest/terraform/coder.tf [hunk 1]
```diff
@@ -34,11 +34,11 @@ resource "kubernetes_namespace" "coder_namespace" {
   ]
 }
 
-resource "random_password" "postgres-admin-password" {
+resource "random_password" "coder-postgres-password" {
```
Comment (johnstcn) [inline scaletest/terraform/coder.tf:37]: review: `postgres-admin-password` was unused

File: scaletest/scaletest.sh [hunk 2]
```diff
@@ -84,11 +84,10 @@ fi
 if [[ -z "${SCALETEST_PROMETHEUS_REMOTE_WRITE_USER}" ]] || [[ -z "${SCALETEST_PROMETHEUS_REMOTE_WRITE_PASSWORD}" ]]; then
 	echo "SCALETEST_PROMETHEUS_REMOTE_WRITE_USER or SCALETEST_PROMETHEUS_REMOTE_WRITE_PASSWORD not specified."
 	echo "No prometheus metrics will be collected!"
-	read -pr "Continue (y/N)? " choice
-	case "$choice" in
-	y | Y | yes | YES) ;;
-	*) exit 1 ;;
-	esac
+	read -p "Continue (y/N)? " -n1 -r
+	if [[ "${REPLY}" != "y" ]]; then
```
Comment (mafredri) [inline scaletest/scaletest.sh:88]:
```suggestion
	if [[ "${REPLY}" != [yY] ]]; then
```

File: scaletest/terraform/prometheus.tf [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-  prometheus_helm_version         = null // just use latest
-  prometheus_release_name         = "prometheus"
-  prometheus_namespace            = "prometheus"
-  prometheus_remote_write_enabled = var.prometheus_remote_write_password != ""
+  prometheus_helm_repo             = "https://charts.bitnami.com/bitnami"
+  prometheus_helm_chart            = "kube-prometheus"
+  prometheus_helm_version          = null // just use latest
+  prometheus_exporter_helm_repo    = "https://prometheus-community.github.io/helm-charts"
+  prometheus_exporter_helm_chart   = "prometheus-postgres-exporter"
+  prometheus_exporter_helm_version = null // use latest
```
Comment (mafredri) [inline scaletest/terraform/prometheus.tf:7]: Just checking; would putting a version here lock it down? If not, perhaps we can exclude it so as to not confuse?

**Review by mafredri — APPROVED** [review]

File: scaletest/terraform/prometheus.tf (hunk 3 above)
Comment (johnstcn) [reply scaletest/terraform/prometheus.tf:7]: Docs say no: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#version

**Pull request merged**